.github/workflows/build.yml

```diff
@@ -105,13 +105,28 @@ jobs:
     runs-on: ubuntu-latest
     needs: [test, lint, examples, license-check]
     if: startsWith(github.ref, 'refs/tags/')
+    env:
+      DOCKER_CLI_EXPERIMENTAL: "enabled"
     steps:
     - uses: actions/checkout@v3
       with:
         fetch-depth: 0
 
     - run: git fetch --force --tags
 
+    - name: Set up QEMU
```

Comment:
Is this `QEMU` step needed? 

Go is able to build binaries for different platforms directly.